src/main/java/org/openstreetmap/atlas/checks/validation/tag/GenericTagCheck.java

```diff
@@ -201,6 +205,23 @@ public GenericTagCheck(final Configuration configuration, final ExternalDataFetc
         }
         this.fetchWikiData(fileFetcher);
         this.fetchTagInfo(fileFetcher);
+
+        /*
+         * Prevent this check from running, unless the user has indicated that not all databases are
+         * required Specifically leave these configuration values undocumented -- if people want to
+         * use only one or the other, they should understand all the issues involved. Namely, they
+         * may have many issues where they are told "key is not documented or popular" even if it is
+         * popular (i.e., they ran with wiki data but no tag info). Regardless, wiki data is
+         * *always* required.
+         */
+        final boolean errorIfDbNotFound = this.configurationValue(configuration, "db.require_all",
```

Comment:
```suggestion
        final boolean errorIfDatabaseNotFound = this.configurationValue(configuration, "database.require_all",
```

Reply:
The first one is done.
The second one I'm not doing (the configuration file already uses `db` as a key), at least until I get feedback from users.

@atiannicelli : Are you OK with updating the `db` key in the configuration file to `database`?